Formula/swtpm.rb

```diff
@@ -0,0 +1,52 @@
+class Swtpm < Formula
+  desc "Software TPM Emulator based on libtpms"
+  homepage "https://github.com/stefanberger/swtpm"
+  url "https://github.com/stefanberger/swtpm/archive/refs/tags/v0.7.3.tar.gz"
+  sha256 "e856d1f5842fb3335164f02f2c545dd329efbc3416db20b7a327e991a4cd49c8"
+  license "BSD-3-Clause"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gawk" => :build
+  depends_on "libtool" => :build
+  depends_on "socat" => :build
+  depends_on "gnutls"
+  depends_on "json-glib"
+  depends_on "libtasn1"
+  depends_on "libtpms"
+  depends_on "openssl@1.1"
+
+  uses_from_macos "expect"
+
+  on_linux do
+    depends_on "libseccomp"
+    depends_on "net-tools"
+  end
+
+  def install
+    args = std_configure_args + [
+      "--with-openssl",
+    ]
+
+    if OS.mac?
+      args << "--with-tss-user=root"
+      args << "--with-tss-group=admin"
+    end
```

Comment:
It really shouldn't try to run/configure as root. Does the tool not work without passing this?

Reply:
There is one package using this user configuration. The default user is `tss` which doesn't exist on OSX. From https://github.com/stefanberger/swtpm/issues/37#issuecomment-340570390, I think we are fine to use the root user.

Reply:
But then the user can only run it as root

Reply:
removed, since it's only used in one of the packages in tests